src/actions/index.js

```diff
@@ -31,10 +31,11 @@ export function fetchLocation(location) {
 
     return geocodeGoogle(location)
       .then((data) => {
-        if (data.status === 'OK') {
+        if (data.status === 'ZERO_RESULTS') {
+          dispatch(changeStatus('Invalid location', 'error'));
+        } else if (data.status === 'OK') {
           return data;
         }
```

Comment:
are there other possible `data.status` messages returned by this API? You may consider a switch statement with a default case for unknown statuses.